### PR TITLE
Form functionality

### PR DESCRIPTION
**Pull request by frank-mck**

### CODE REVIEW

File: client/db.json [hunk 1]
```diff
@@ -8,53 +8,25 @@
       "id": 0
     },
     {
-      "lng": "-3.869932960937492",
-      "lat": "55.4045715267554",
-      "title": "uigp",
-      "description": "uy9",
+      "text": "fdbdbd",
```
Comment (YinnyF) [inline client/db.json:11]: would we be able to retain the naming convention to "title"?

Comment (frank-mck) [reply client/db.json:11]: On it

File: client/src/components/maps/MyMap.js [hunk 2]
```diff
@@ -81,10 +66,11 @@ export const MyMap = () => {
         styles: mapStyle,
         disableDefaultUI: true,
         zoomControl: true,
+        minZoom: 4,
+        maxZoom: 18,
```
Comment (YinnyF) [inline client/src/components/maps/MyMap.js:70]: i like it

**Review by YinnyF — APPROVED** [review]
great work

**Pull request merged**